sdk/version.go

```diff
@@ -68,5 +68,5 @@ const (
 
 // GetCoreOSAge returns the number of days since the CoreOS epoch.
 func GetCoreOSAge() int64 {
-	return int64(time.Since(time.Unix(CoreOSEpoch, 0)).Seconds()) / 86400
+	return int64(time.Since(time.Unix(CoreOSEpoch, 0)) / (86400 * time.Second))
```

Comment:
Er, wait. nm

I got my parrens mismatched, you are doing that. this is why I don't write LISP